src/components/Personal-Zone/Publisher/empty-display/EmptyDisplay.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+
+import Typography from '@material-ui/core/Typography';
+
+import styles from './EmptyDisplayStyles';
+import NewJobButton from '../new-job/new-job-button/NewJobButton';
+import ResumeIcon from '../../../../assets/icons/ResumeIcon';
+
+const EmptyPublisherDisplay: React.FC= () => {
+    const classes = styles();
+
+    return (
+        <div className={classes.root}>
+            <ResumeIcon className={classes.resumeIcon}/>
+            <span className={classes.messageTitle}>
+                פרסמת? מצאת!
+            </span>
+            <Typography variant='body1' className={classes.messageBody}>
```

Comment:
I would use <br/> to break lines, as this can be done in 1 Typography. Also, it's almost like the PowerPoint (Some lines got broke...)

But this it not so important.. can be done in future 😃

Reply:
`גבוה ביותר`

Same row 😁

Reply:
the line breaks here are just for readability (html does not notice them(